src/SFA.DAS.AssessorService.Api.Types/Models/Register/UpdateEpaOrganisationContactRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using MediatR;
+
+namespace SFA.DAS.AssessorService.Api.Types.Models.Register
+{
+    public class UpdateEpaOrganisationContactRequest : IRequest<string>
```

Comment:
Just wondering if this be better if it returned a Contact object or an UpdateEpaOrganisationContactResponse than a string

Reply:
I don't need the contact returned, just the id of the contact to work with the MVC stateless flow.  There's a separate endpoint to get the contact from the id returned, if needed.